rayon-core/src/registry.rs

```diff
@@ -924,6 +917,20 @@ unsafe fn main_loop(thread: ThreadBuilder) {
     }
 }
 
+unsafe fn wait_until_out_of_work(worker_thread: &WorkerThread) {
```

Comment:
Could this be a `&self` method instead? (next to the existing `wait_until` methods)

Reply:
Yeah, that works and is nicer / more in-line with the other methods indeed.